types/react-autosuggest/react-autosuggest-tests.tsx

```diff
@@ -89,7 +91,7 @@ export class ReactAutosuggestBasicTest extends React.Component<any, any> {
             sectionTitle: { color: 'blue' }
         };
 
-        return <Autosuggest
+        return <LanguageAutosuggest
```

Comment:
If possible, we should leave this as is to test backward compatibility and duplicate it to test the `LanguageAutosuggest` version

Reply:
Left the `ReactAutosuggestBasicTest` for backward compatibility, added `ReactAutosuggestTypedTest` which is identical but strongly typed.